localisation/00_zuwc_united_localisation.csv

```diff
@@ -59,7 +59,7 @@ EVTOPTA_WCCLS_2100;I'll help you.;;Ich werde Euch helfen.;;;;;;;;;;;x
 EVTOPTB_WCCLS_2100;I can help you for a price.;;Ich kann Euch helfen, f�r etwas Gold nat�rlich.;;;;;;;;;;;x
 EVTOPTC_WCCLS_2100;No, [From.GetSheHe] isn't worth my attention.;Non, [From.GetIlElle] ne m�rite pas mon attention.;Nein, [FromFrom.GetSheHe] ist meiner Aufmerksamkeit nicht w�rdig.;;;;;;;;;;;x
 EVTOPTB_WCCLS_2102;It's too expensive for me.;;Das ist zu kostspielig.;;;;;;;;;;;x
-EVTDESC_WCFTW_101;As the great war between the Orcish Horde and the kingdom of Stormwind rages on, the human kingdom have lost much against the orcish invaders. Their envoys at our court now plead for our aid. How shall we answer?;;W�hrend der gro�e Krieg zwischen der Orkhorde und dem K�nigreich von Stormwind tobt, haben die Menschen Stormwinds viel an die Orks verloren. Ihre Gesandten an unserem Hof pl�dieren jetzt f�r unsere Hilfe, wie sollen wir antworten?;;;;;;;;;;;x
+EVTDESC_WCFTW_101;As the great war between the orcish Horde and the kingdom of Stormwind rages on, the human kingdom has lost much against the orcish invaders. Their envoys at our court now plead for our aid. How shall we answer?;;W�hrend der gro�e Krieg zwischen der Orkhorde und dem K�nigreich von Stormwind tobt, haben die Menschen Stormwinds viel an die Orks verloren. Ihre Gesandten an unserem Hof pl�dieren jetzt f�r unsere Hilfe, wie sollen wir antworten?;;;;;;;;;;;x
```

Comment:
Orcish is a part of the name.
https://wow.gamepedia.com/Old_Horde

Reply:
ok, done